pkg/virt-handler/vm.go

```diff
@@ -291,6 +292,12 @@ func (d *VMHandlerDispatch) processVmUpdate(vm *v1.VM, shouldDeleteVm bool) erro
 		return err
 	}
 
+	// Map Container Registry Disks to block devices Libvirt can consume
+	vm, err = registrydisk.MapRegistryDisks(vm)
+	if err != nil {
+		return err
+	}
```

Comment:
> There are countless different ways to do dynamic storage and this docker image is just one possibility

It is one, which is implicitly brought to us by using k8s. It is a little bit like PVs and PVCs. Whenever you deal with k8s, you either have a public or a private docker registry at your disposal.